vars/filebeat.groovy

```diff
@@ -36,7 +36,13 @@ def call(Map args = [:], Closure body) {
 }
 
 def start(Map args = [:]) {
-  def output = args.containsKey('output') ? args.output : 'docker_logs.log'
+  def sanitizedOuput = ''
+  if (args.containsKey('output')) {
+    sanitizedOuput = sanitizeOutputFileName(args.output)
+  }
+  sanitizedOuput = sanitizedOuput?.trim() ? sanitizedOuput : 'docker_logs.log'
+
+  def output = sanitizedOuput
```

Comment:
```suggestion
  def sanitizedOutput = ''
  if (args.containsKey('output')) {
    sanitizedOutput = sanitizeOutputFileName(args.output)
  }
  sanitizedOutput = sanitizedOutput?.trim() ? sanitizedOutput : 'docker_logs.log'

  def output = sanitizedOutput
```

Reply:
I consistently made the typo! good catch!